frontends/web/src/routes/account/account.tsx

```diff
@@ -72,7 +73,7 @@ class Account extends Component<Props, State> {
     public readonly state: State = {
         initialized: false,
         connected: false,
-        transactions: undefined,
+        transactions: undefined, // TBD: default to empty array: []
```

Comment:
comment might be wrong. empty list of transactions is different than not having the transactions list available at all.s 
maybe this doesn't happen though, good to check.

Reply:
nono you are correct, but I'd change it everywhere to always be an array and never undefined. I removed the comment and might make a PR for it another time.